cmd/utils/flags.go

```diff
@@ -280,640 +306,736 @@ var (
+		Usage:    "Subscribes blocks from redis trie node cache",
+		Aliases:  []string{},
+		EnvVars:  []string{"KLAYTN_STATEDB_CACHE_REDIS_SUBSCRIBE"},
+		Category: "CACHE",
 	}
 	TrieNodeCacheLimitFlag = &cli.IntFlag{
-		Name:    "state.trie-cache-limit",
-		Usage:   "Memory allowance (MiB) to use for caching trie nodes in memory. -1 is for auto-scaling",
-		Value:   -1,
-		Aliases: []string{},
-		EnvVars: []string{"KLAYTN_STATE_TRIE_CACHE_LIMIT"},
+		Name:     "state.trie-cache-limit",
+		Usage:    "Memory allowance (MiB) to use for caching trie nodes in memory. -1 is for auto-scaling",
+		Value:    -1,
+		Aliases:  []string{},
+		EnvVars:  []string{"KLAYTN_STATE_TRIE_CACHE_LIMIT"},
+		Category: "CACHE",
 	}
 	TrieNodeCacheSavePeriodFlag = &cli.DurationFlag{
-		Name:    "state.trie-cache-save-period",
-		Usage:   "Period of saving in memory trie cache to file if fastcache is used, 0 means disabled",
-		Value:   0,
-		Aliases: []string{},
-		EnvVars: []string{"KLAYTN_STATE_TRIE_CACHE_SAVE_PERIOD"},
+		Name:     "state.trie-cache-save-period",
+		Usage:    "Period of saving in memory trie cache to file if fastcache is used, 0 means disabled",
+		Value:    0,
+		Aliases:  []string{},
+		EnvVars:  []string{"KLAYTN_STATE_TRIE_CACHE_SAVE_PERIOD"},
+		Category: "CACHE",
 	}
 
 	SenderTxHashIndexingFlag = &cli.BoolFlag{
-		Name:    "sendertxhashindexing",
-		Usage:   "Enables storing mapping information of senderTxHash to txHash",
-		Aliases: []string{"common.sender-tx-hash-indexing"},
-		EnvVars: []string{"KLAYTN_SENDERTXHASHINDEXING"},
+		Name:     "sendertxhashindexing",
+		Usage:    "Enables storing mapping information of senderTxHash to txHash",
+		Aliases:  []string{"common.sender-tx-hash-indexing"},
+		EnvVars:  []string{"KLAYTN_SENDERTXHASHINDEXING"},
+		Category: "DATABASE",
 	}
 	ChildChainIndexingFlag = &cli.BoolFlag{
-		Name:    "childchainindexing",
-		Usage:   "Enables storing transaction hash of child chain transaction for fast access to child chain data",
-		Aliases: []string{"common.child-chain-indexing"},
-		EnvVars: []string{"KLAYTN_CHILDCHAININDEXING"},
+		Name:     "childchainindexing",
+		Usage:    "Enables storing transaction hash of child chain transaction for fast access to child chain data",
+		Aliases:  []string{"common.child-chain-indexing"},
+		EnvVars:  []string{"KLAYTN_CHILDCHAININDEXING"},
+		Category: "SERVICECHAIN",
 	}
 	TargetGasLimitFlag = &cli.Uint64Flag{
-		Name:    "targetgaslimit",
-		Usage:   "Target gas limit sets the artificial target gas floor for the blocks to mine",
-		Value:   params.GenesisGasLimit,
-		Aliases: []string{"common.target-gaslimit"},
-		EnvVars: []string{"KLAYTN_TARGETGASLIMIT"},
+		Name:     "targetgaslimit",
+		Usage:    "Target gas limit sets the artificial target gas floor for the blocks to mine",
+		Value:    params.GenesisGasLimit,
+		Aliases:  []string{"common.target-gaslimit"},
+		EnvVars:  []string{"KLAYTN_TARGETGASLIMIT"},
+		Category: "NETWORK",
```

Comment:
```suggestion
		Category: "CONSENSUS",
```

Reply:
This flag is actually accessed setupNetwork() function. I think that's why this flag has in NETWORK flagGroup.
Actually as you mentioned, It's not clear a bit. How do you think the refactoring these things as next step?